cli/pkg/utils/utils_test.go

```diff
@@ -0,0 +1,68 @@
+package utils
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestExportValuesFieldValue(t *testing.T) {
+	type args struct {
+		chart string
+		keys  []string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    interface{}
+		wantErr bool
+	}{
+		{
+			name: "Success test-case to fetch nothing as the provided keys are empty",
+			args: args{
+				chart: "../../../chart/",
+				keys:  nil,
+			},
+			want: nil,
+			wantErr: false,
```

Comment:
I would omit `wantErr` from here since by default its false, easier to scan for tests which return errors.

I would personally do the same for `want` since default is nil.